Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6198,8 +6198,7 @@ def com_google_fonts_check_colorfont_tables(ttFont):
         Having a glyph with no contours on GID 1 is a practical workaround for that.
 
         See https://github.com/googlefonts/gftools/issues/609
-    """,
-    proposal = 'https://github.com/googlefonts/fontbakery/issues/3886'
+    """
```

Comment:
In this case, it is better to replace the old/incorrect proposal URL by `proposal = 'https://github.com/googlefonts/gftools/issues/609'`

Reply:
It would be even better to list these 2 relevant URLs, since that's where people would find the whole story of how the new check was initially proposed, drafted and reviewed:

```python
proposal = ['https://github.com/googlefonts/gftools/issues/609',
            'https://github.com/googlefonts/fontbakery/pull/3905']
```